### ISSUE TITLE
Edit instructions for VA 508 Office support to include Martha

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34616]
## Issue Description
The [How to request support from the VA 508 Office](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-request-support-from-the-VA-508-Office.1857913061.html) instructions on Platform Website need to be modified so that the OCTO Accessibility Lead, Martha Wilkes, is included. The instructions should be edited to instruct the VFS team member to include Martha's name in the "Requested for" field so that she is cc'd on all requests. Under the "Requested for" phone number location, the instructions should instruct the VFS team member to list "VACO" as the location, with Martha's phone number (available in her Slack profile*).

*Note - do not list Martha's phone number in the How to request support from the VA 508 Office instructions as this is a public site that anyone can access. The instructions should link to Martha's profile in Slack and VFS team members can get it there. 

---
## Tasks
- [x] Create a draft page under Documentation Drafts
- [x] Add steps to the instructions for populating the "Requested for" and "Requested for phone number" to the [How to request support from the VA 508 Office](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-request-support-from-the-VA-508-Office.1857913061.html) Audit Request instructions on Platform Website. Currently step 6 covers how to fill out the form. 
   - _Before editing, it is advised that you should follow the instructions yourself as if you were submitting your own audit request. You can [reference this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/29924) to view a gif of what the form should look like._ 
- [x] Peer review with a Product Support teammate
- [x] Share draft page with Martha (in slack in accessibility-ops or platform team) for review and approval

## Acceptance Criteria
- [x] Platform Website guidance has been updated
- [ ] Team has reviewed/approved the changes

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by briandeconinck** [comment]
@shiragoodman Just to confirm, these edits for only the [Audit Request section](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-request-support-from-the-VA-508-Office.1857913061.html#HowtorequestsupportfromtheVA508Office-AuditRequest) at step 6 -- correct? Or do changes need to be made to the Document Review section as well?

**Comment by shiragoodman** [comment]
@briandeconinck correct, just the Audit request section. thanks for checking!

**Comment by briandeconinck** [comment]
[Draft version](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2041479217/Brian+-+Copy+of+How+to+request+support+from+the+VA+508+Office) of the updated page complete.

**Comment by shiragoodman** [comment]
@briandeconinck I made a couple comments in your draft. Please make the requested changes then share with Martha per the guidance above. thanks!
